collector/mssql.go

```diff
@@ -396,13 +396,13 @@ func NewMSSQLCollector() (Collector, error) {
 		// meta
 		mssqlScrapeDurationDesc: prometheus.NewDesc(
 			prometheus.BuildFQName(Namespace, subsystem, "collector_duration_seconds"),
-			"wmi_exporter: Duration of an mssql child collection.",
+			"windows_exporter: Duration of an mssql child collection.",
```

Comment:
Do we need the prefix here at all? It seems other collectors doesnt have a "wmi_exporter:" prefix.

Reply:
I _think_ the idea is to distinguish that this is a metric about the exporter itself, rather than the monitored system? :shrug: